chainntnfs/txconfnotifier.go

```diff
@@ -150,14 +150,23 @@ const (
 	rescanComplete
 )
 
+// confNtfnSet holds all known, registered confirmation notifications for a
+// single txid. If duplicates notifications are requested, only one historical
+// dispatch will be spawned to ensure redundant scans are not permitted. A
+// single conf detail will be constructed and dispatched to all interested
+// clients.
```

Comment:
Squash with the commit introducing `confNtfnSet` :)

Reply:
Ok, several more "fixup" commits that are not marked fixup so will not comment on them for now :)